benchmark/data/tweet.json

```diff
@@ -0,0 +1,66 @@
+{
+  "text": "@wildfits you're not getting one.....",
```

Comment:
Now I'm wondering how you ended with [this tweet](https://twitter.com/byjessiekay/status/22824602300).

Reply:
The file is from the benchmark specification data, where it was added in https://github.com/mongodb/specifications/commit/19ec68039b12f431d30848203c7ccd55815b9319. It probably ended up in a test dataset at some point in time and just kept on being copied around since then.